README.md

```diff
@@ -45,7 +42,6 @@ After you're finished please remove all the comments and instructions!
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
```

Comment:
- [ ] [optional] @MosDevx Great work on your documentation, however, it would have been nice if you added key features and future features sections.